src/image-header.js

```diff
@@ -12,6 +12,11 @@ const APP_MARKER_SIZE = 2;
 const TIFF_HEADER_OFFSET = 10; // From start of APP1 marker.
 const IPTC_DATA_OFFSET = 18; // From start of APP13 marker.
 const XMP_DATA_OFFSET = 33; // From start of APP1 marker.
+const APP2_ICC_DATA_OFFSET = 18; // From start of APP2 marker including marker and chunk/chunk total numbers
+
+const APP2_ICC_IDENTIFIER = 'ICC_PROFILE\0';
+const ICC_CHUNK_NUMBER_OFFSET = APP_MARKER_SIZE + 2 + APP2_ICC_IDENTIFIER.length;
```

Comment:
What is 2? Could you add another constant for it?

Reply:
Changed to APP_ID_OFFSET